src/utils/generateDisplayName.js

```diff
@@ -4,5 +4,5 @@ import isTag from './isTag';
 import type { Target } from '../types';
 
 export default function generateDisplayName(target: Target): string {
-  return isTag(target) ? `styled.${target}` : `Styled(${getComponentName(target)})`;
+  return isTag(target) ? `styled.${getComponentName(target)}` : `Styled(${getComponentName(target)})`;
```

Comment:
why was this change added? it's not desired

Reply:
Flow can't coerce correctly to string type for some reason

Reply:
Ah ok, just put a `// $FlowFixMe`

Reply:
done